xla/service/gpu/ir_emission_utils.h

```diff
@@ -51,7 +51,7 @@ inline constexpr int64_t WarpSize() { return 32; }
 
 // Need at least 1024 threads/block for reasonable tree reduction
```

Comment:
The comment should also be updated to reflect the code change.

Reply:
Right. I made this PR quickly just to help Google test it internally.
I just updated it to be sure it isn't forgotten.